Articles/HowTos/CriticalBeginnerGitUsageTips.md

```diff
@@ -1,11 +1,13 @@
 # Critical Beginner Git Usage Tips
 
 <!-- deck text start -->
-Follow these simple guidelines for setting up and using Git and you'll stay out of serious trouble and be more productive.
+This article describes simple guidelines for setting up and using Git.
```

Comment:
@rinkug, why this change made?  Is this not subjective to whether this is a better intro or not?